collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/SelectMultiMinimalDialog.java

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.fragments.dialogs;
+
+import android.os.Bundle;
+import android.view.View;
+
+import org.javarosa.core.model.data.helper.Selection;
+import org.jetbrains.annotations.NotNull;
+import org.odk.collect.android.adapters.AbstractSelectListAdapter;
+import org.odk.collect.android.adapters.SelectMultipleListAdapter;
+import org.odk.collect.android.listeners.SelectItemClickListener;
+import org.odk.collect.android.logic.ChoicesRecyclerViewAdapterProps;
+
+import java.util.List;
+
+public class SelectMultiMinimalDialog extends SelectMinimalDialog implements SelectItemClickListener {
+    private SelectMultipleListAdapter adapter;
+
+    public SelectMultiMinimalDialog() {
+    }
+
+    public SelectMultiMinimalDialog(List<Selection> selectedItems, boolean isFlex, boolean isAutoComplete, ChoicesRecyclerViewAdapterProps props) {
+        super(isFlex, isAutoComplete);
+        adapter = new SelectMultipleListAdapter(selectedItems, this, props);
+    }
+
+    @Override
+    public void onViewCreated(@NotNull View view, Bundle savedInstanceState) {
```

Comment:
I don't think this is needed.

Reply:
Why? Have you seen the comment below?

Reply:
Ahh I see because we don't use it either way right right...